### PR TITLE
Feat/hf classifier autoconfig

### PR DESCRIPTION
**Pull request by vaskonov**

### CODE REVIEW

File: deeppavlov/configs/classifiers/insults_kaggle_bert_torch.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
     "pipe": [
       {
         "class_name": "torch_bert_preprocessor",
```
Comment (yurakuratov) [inline deeppavlov/configs/classifiers/insults_kaggle_bert_torch.json:21]: torch_bert_preprocessor -> torch_transformers_preprocessor

File: deeppavlov/configs/classifiers/insults_kaggle_bert_torch.json [hunk 2]
```diff
@@ -59,7 +59,7 @@
         "class_name": "torch_bert_classifier",
```
Comment (yurakuratov) [inline deeppavlov/configs/classifiers/insults_kaggle_bert_torch.json:59]: torch_bert_preprocessor -> torch_transformers_preprocessor

**Review by yurakuratov — COMMENTED** [review]
I think that it is needed to check which pre-trained models from this list https://huggingface.co/transformers/pretrained_models.html
are working with DeepPavlov classification pipeline and at least specify supported and unsupported models.

UPD: I have tried to use DialoGPT-small and got
```
ValueError: Unrecognized configuration class <class 'transformers.configuration_gpt2.GPT2Config'> for this kind of AutoModel: AutoModelForSequenceClassification.
Model type should be one of DistilBertConfig, AlbertConfig, CamembertConfig, XLMRobertaConfig, BartConfig, RobertaConfig, BertConfig, XLNetConfig, FlaubertConfig, XLMConfig
```
So, this is that list of models that could be used for classification.

File: deeppavlov/configs/classifiers/insults_kaggle_bert_torch.json (hunk 1 above)
Comment (vaskonov) [reply deeppavlov/configs/classifiers/insults_kaggle_bert_torch.json:21]: maybe a single better than plural?
torch_transformer_preprocessor

Comment (vaskonov) [reply deeppavlov/configs/classifiers/insults_kaggle_bert_torch.json:21]: @IgnatovFedor we need your opinion here

Comment (yurakuratov) [reply deeppavlov/configs/classifiers/insults_kaggle_bert_torch.json:21]: When I wrote `transformers` I thought about HuggingFace Transformers, not about NN architecture. Don't know what is better here.

Comment (IgnatovFedor) [reply deeppavlov/configs/classifiers/insults_kaggle_bert_torch.json:21]: torch_transformers_preprocessor, agree with @dilyararimovna

**Review by IgnatovFedor — APPROVED** [review]

**Pull request merged**